code/modules/mob/living/simple_animal/hostile/hivebot.dm

```diff
@@ -33,14 +33,14 @@
 
 /mob/living/simple_animal/hostile/hivebot/range
 	name = "Ranged Hivebot"
-	desc = "A smallish robot, this one has a gun!"
+	desc = "A biggish robot, this one has a gun!"
```

Comment:
COMMA SPLICE

Applies to the next one, too

Sorry for making this a separate review; I didn't notice this before.